src/core/expression/qgsexpression.cpp

```diff
@@ -771,6 +779,9 @@ void QgsExpression::initVariableHelp()
 
   //provider notification
   sVariableHelpTexts.insert( QStringLiteral( "notification_message" ), QCoreApplication::translate( "notification_message", "Content of the notification message sent by the provider (available only for actions triggered by provider notifications)." ) );
+
+  //form context variable
+  sVariableHelpTexts.insert( QStringLiteral( "current_geometry" ), QCoreApplication::translate( "current_geometry", "Represents the geometry of the feature currently being edited in the form or the table row. Can be used for in a form/row context to filter the related features." ) );
```

Comment:
You'll also need to add current_feature here

Reply:
"Can be used **for** in a form/row context to filter the related features."
Is that a correct sentence?

Reply:
fixed in master